pkg/apis/serving/v1alpha2/transformer.go

```diff
@@ -51,6 +51,9 @@ func (t *TransformerSpec) Validate(config *InferenceServicesConfig) error {
 	if err != nil {
 		return err
 	}
+	if err := validateResourceRequirements(&transformer.GetContainerSpec().Resources); err != nil {
```

Comment:
and here.

Reply:
Looking at the spec, I think we should probably be validating min/max replicas like we do in predictor. Is there a way we can generalize this?

Reply:
OK, I will do this validating min/max replicas for transformer and explainer too.

Reply:
Logged issue #495 for this.